File: Chalkable.BusinessLogic/Model/AcademicBenchmark/GradeLevel.cs
namespace Chalkable.BusinessLogic.Model.AcademicBenchmark
{
    public class GradeLevel
    {
        public string Code { get; set; }
        public string Description { get; set; }
        public string Low { get; set; }
        public string Height { get; set; }
        public static GradeLevel Create(AcademicBenchmarkConnector.Models.GradeLevel grLevel)
        {
            return new GradeLevel
            {
                Description = grLevel.Description,
                Code = grLevel.Code,
                Height = grLevel.Height,
                Low = grLevel.Low
            };
        }
    }
}

File: Chalkable.Web/app/chlk/models/academicBenchmark/GradeLevel.js
REQUIRE('chlk.models.id.ABSubjectDocumentId');
REQUIRE('chlk.models.id.ABAuthorityId');
REQUIRE('chlk.models.id.ABDocumentId');

NAMESPACE('chlk.models.academicBenchmark', function(){

    /**@class chlk.models.academicBenchmark.GradeLevel*/

    CLASS('GradeLevel', [

        String, 'code',
        String, 'description',
        String, 'low',
        String, 'height',
        chlk.models.id.ABAuthorityId, 'authorityId',
        chlk.models.id.ABDocumentId, 'documentId',
        chlk.models.id.ABSubjectDocumentId, 'subjectDocumentId',

        function getName(){
            return this.getCode();
        },

        function getId(){
            return this.getCode();
        }
    ]);
});
